pkg/api/customization/project/project_actions.go

```diff
@@ -137,13 +137,15 @@ func (h *Handler) viewMonitoring(actionName string, action *types.Action, apiCon
 	}
 
 	// need to support `map[string]string` as entry value type in norman Builder.convertMap
-	answers, err := convert.EncodeToMap(monitoring.GetOverwroteAppAnswers(project.Annotations))
+	answers, version := monitoring.GetOverwroteAppAnswersAndVersion(project.Annotations)
+	encodeAnswers, err := convert.EncodeToMap(answers)
 	if err != nil {
 		return httperror.WrapAPIError(err, httperror.ServerError, "failed to parse response")
 	}
 	apiContext.WriteResponse(http.StatusOK, map[string]interface{}{
-		"answers": answers,
+		"answers": encodeAnswers,
 		"type":    "monitoringOutput",
+		"version": version,
```

Comment:
Maybe you should not return `version` when it is empty string.

Reply:
updated.